external-crates/move/crates/move-compiler/src/parser/ast.rs

```diff
@@ -1849,7 +1852,11 @@ impl AstDebug for PathEntry {
 
 impl AstDebug for NamePath {
     fn ast_debug(&self, w: &mut AstWriter) {
-        let NamePath { root, entries } = self;
+        let NamePath {
+            root,
+            entries,
+            incomplete: _,
+        } = self;
         w.write(format!("{}::", root));
         w.list(entries, "::", |w, e| {
             e.ast_debug(w);
```

Comment:
Indicate if it's incomplete here. Maybe print a trailing `::_` or something?

Reply:
Done